docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -39,12 +32,9 @@ controller-2   192.168.24.38  _admin mon mgr
 ----
 
 ifeval::["{build}" != "downstream"]
-. During the overcloud deployment, RGW is applied at step 2
-(external_deployment_steps), and a cephadm compatible spec is generated in
-`/home/ceph-admin/specs/rgw` from the https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/ansible_plugins/modules/ceph_mkspec.py[ceph_mkspec] ansible module.
-Find and patch the RGW spec, specifying the right placement using the labels
-approach, and change the rgw backend port to 8090 to avoid conflicts
-with the https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/roles/tripleo_cephadm/tasks/rgw.yaml#L26-L30[Ceph Ingress Daemon] (*)
+. During the overcloud deployment, a cephadm compatible spec is generated in
+`/home/ceph-admin/specs/rgw`. Find and patch the RGW spec, specifying the right placement using the labels
+approach, and change the rgw backend port to 8090 to avoid conflicts with the Ceph Ingress daemon frontend port.
```

Comment:
```suggestion
. During the overcloud deployment, a `cephadm`-compatible spec is generated in
`/home/ceph-admin/specs/rgw`. Find and patch the RGW spec, specify the right placement by using labels, and change the RGW back-end port to `8090` to avoid conflicts with the Ceph Ingress daemon front-end port.
```

Specify the right placement of the nodes? If yes, we should add that. E.g., "...specify the right placement of the nodes by using labels..."

Reply:
Yes, it works for me, I'm pushing a new change for that incorporating your suggestion.